Library/Formula/synergy.rb

```diff
@@ -0,0 +1,75 @@
+class Synergy < Formula
+  homepage 'http://synergy-project.org'
+  url 'https://github.com/synergy/synergy/archive/1.6.3.tar.gz'
+  sha1 'c4c26ee983f0684a5cbe45520adc0b50a71a3524'
```

Comment:
:+1:

sorry for the error: I have recycled an older version of the formula, fixing that.